spotipy/client.py

```diff
@@ -405,6 +405,10 @@ def playlist_tracks(self, playlist_id, fields=None,
         return self._get("playlists/%s/tracks" % (plid),
                          limit=limit, offset=offset, fields=fields,
                          market=market)
+    
+    def playlist_cover(self, playlist_id=None):
```

Comment:
A comment under the method (see method below) will help to automatically generate documentation for that method on https://spotipy.readthedocs.io/

Reply:
edited